spec/jwt_spec.rb

```diff
@@ -1,434 +1,502 @@
-require 'helper'
+require 'spec_helper'
+require 'jwt'
 
 describe JWT do
-  before do
-    @payload = {'foo' => 'bar', 'exp' => Time.now.to_i + 1, 'nbf' => Time.now.to_i - 1 }
+  let :payload do
+    {
+      user_id: 'some@user.tld'
+    }
+  end
+
+  let :data do
```

Comment:
given that this isn't overridden, it might be worth making this a constant instead of a let block.

as-is, each of those files will be re-read for any it block that accesses data